js/src/edit-block/components/settings/checkbox.js

```diff
@@ -28,7 +29,6 @@ const Checkbox = ( { handleOnChange, setting, value } ) => {
 				id={ id }
 				className="ml-2"
 				type="checkbox"
-				value={ checked }
```

Comment:
It [looks like](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/Input/checkbox) this isn't required